apps/registry/helm-release.yaml

```diff
@@ -12,40 +11,35 @@ spec:
         kind: HelmRepository
         name: registry
       interval: 1h
-
   values:
     ingress:
       enabled: true
       className: traefik
       annotations:
         cert-manager.io/cluster-issuer: letsencrypt-prod
       hosts:
-      - registry.chaosdorf.space
+        - registry.chaosdorf.space
       paths:
-      - /
+        - /
       tls:
-      - secretName: ingress-tls
-        hosts:
-        - registry.chaosdorf.space
-
+        - secretName: ingress-tls
+          hosts:
+            - registry.chaosdorf.space
     resources:
       limits:
         cpu: 100m
         memory: 128Mi
       requests:
         cpu: 10m
         memory: 32Mi
-
     persistence:
       enabled: true
       size: 20Gi
-      storageClass: 'ceph-block'
-
+      storageClass: ceph-block
     metrics:
       enabled: true
       port: 5001
-
     garbageCollect:
       enabled: true
       deleteUntagged: true
-      schedule: "0 1 * * *"
+      schedule: 0 1 * * *
```

Comment:
mindestens das hier muss wieder rückgängig gemacht werden, zumindest nach dem Beispiel in https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/ scheint das keine valide Syntax zu sein.